CMakeLists.txt

```diff
@@ -364,7 +364,7 @@ set(LINUX_LDFLAGS -target ${LLVM_TARGET}
                   -l:libc.a
                   -static)
 
-set(COMMON_LINUX_LINK_LIBRAIRIES configfile)
+set(COMMON_LINUX_LINK_LIBRARIES configfile)
```

Comment:
This looks like the right change, but probably better as its own PR.

Reply:
- #12958